resolves #63 fix up VI and Describe help pages

1jamesthompson1 opened the pull request:
Merging VI with VI.ggplot and creating a new Describe help page.

1jamesthompson1 committed:
resolves #63

Merging VI with VI.ggplot and creating a new Describe help page.

1jamesthompson1 commented:
Oh, no will have a look at it thought it was building alright.

ajrgodfrey commented:
Doing likewise.
J


From: 1jamesthompson1 ***@***.***>
Sent: Tuesday, 6 December 2022 4:31 pm
To: ajrgodfrey/BrailleR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [ajrgodfrey/BrailleR] resolves #63 fix up VI and Describe help pages (PR #66)


Oh, no will have a look at it thought it was building alright.

-
Reply to this email directly, view it on GitHub<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fajrgodfrey%2FBrailleR%2Fpull%2F66%23issuecomment-1338692210&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C448c9ecbf462474654ee08dad73a4a01%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058942622026376%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IB1UU%2FnrcpDhXmOAcmeCkvab97FxjJJ3ouauNaOPtwc%3D&reserved=0>, or unsubscribe<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FADCZKGXYZKSKVKXBANXMXZDWL2XO7ANCNFSM6AAAAAASU77SVU&data=05%7C01%7Ca.j.godfrey%40massey.ac.nz%7C448c9ecbf462474654ee08dad73a4a01%7C388728e1bbd0437898dcf8682e644300%7C1%7C0%7C638058942622026376%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Ux5mdr0%2Bi4OCyFWKn9g%2BusWI2YMJ3ea854qVP0E2V9M%3D&reserved=0>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

1jamesthompson1 committed:
fix build error

Forgot to add a check to make sure that ggplot package is loaded in the examples section.

Also fixed some warnings and notes.
